README.md

```diff
@@ -1375,6 +1375,12 @@ Something is missing?  Create a pull request!
         [github](https://github.com/tarent)
         ,
         [jobs](https://www.tarent.de/en/jobs/)
+    * [Technologiestiftung Berlin / CityLAB](https://www.technologiestiftung-berlin.de) - 
+        [proof](https://github.com/search?q=org%3Atechnologiestiftung+django&type=code)
```

Comment:
True! I had not checked the results to be honest... 

We had actually old projects based on Django that were archived and we use Django for projects we develop with partner, like https://github.com/KWB-R/qmra but that won't show out in the list.

We will very probably use Django soon in a new project, which is also why I wanted to open the PR.

However, I understand this is not a proof. What shall we do?

Reply:
@MRigal maybe I have an idea. Given that you work at that place and hence have "some write permissions" you could potentially produce _new_ proof to link to, correct? For instance, [job ad Python Backend Developer:in](https://www.technologiestiftung-berlin.de/fileadmin/Redaktion/PDFs/Stellenanzeigen/2022/TSB-Stellenausschreibung-DigitalServices-Backend-Python-20220930.pdf) currently reads…

> Du hast sehr gute Kenntnisse in Python und SQL-Datenbanken sowie von
gängingen Frameworks (Django, Flask, FastAPI ...)

…which does not say whether you have any of Django, Flask, FastAPI today. BUT if the ad had a sentence more on the technologies that you are using already today — the current tech stack or a fraction, as long as Django is mentioned — that would be proof enough in the context of this list. Does that sounds fair and practical? I'm open to other ideas, but that's the best I have right now. What do you think?

PS: A page on https://stackshare.io/ would also work but it's not free of cost, afaik.

Reply:
No I can't change the job ad alone, and also it will probably be removed soon... 

But thanks for the stackshare hint! We are currently working on a tech radar, that we plan to launch soon, I'll keep the PR open until I can link it :-)

Reply:
> But thanks for the stackshare hint! We are currently working on a tech radar, that we plan to launch soon,

Tech radar as in https://opensource.zalando.com/tech-radar/ ?

>I'll keep the PR open until I can link it :-)

Deal :smile: :+1: